package.json

```diff
@@ -84,27 +85,203 @@
     }
   },
   "contributes": {
+    "icons": {
+      "camel-run": {
+        "description": "Run Camel Application",
+        "default": {
+          "fontPath": "./icons/camel-run.woff",
+          "fontCharacter": "\\E900"
+        }
+      }
+    },
     "commands": [
       {
         "command": "kaoto.open",
-        "title": "Open with Kaoto Graphical Editor for Camel"
+        "title": "Open with Kaoto"
       },
       {
-        "command": "kaoto.open.textualeditor",
-        "title": "Open Camel file with textual editor on the side",
-        "shortTitle": "Show source",
+        "command": "kaoto.open.source",
+        "title": "Open Source Code",
+        "shortTitle": "Show Source",
         "enablement": "activeCustomEditorId == webviewEditorsKaoto",
-        "category": "Camel",
-        "icon": "$(go-to-file)"
+        "category": "Kaoto",
+        "icon": "$(code)"
+      },
+      {
+        "command": "kaoto.close.source",
+        "title": "Close Source Code",
+        "shortTitle": "Close Source",
+        "enablement": "resourceFilename =~ /\\.(yaml)$/",
+        "category": "Kaoto",
+        "icon": "$(code)"
+      },
+      {
+        "command": "kaoto.new.file",
+        "title": "Camel File",
+        "category": "Kaoto",
+        "enablement": "false",
+        "icon": "$(new-file)"
+      },
+      {
+        "command": "kaoto.camel.jbang.export",
+        "title": "Export",
+        "category": "Kaoto",
+        "icon": "$(link-external)"
+      },
+      {
+        "command": "kaoto.camel.jbang.routes.yaml",
+        "shortTitle": "Create a Route",
+        "title": "Create a Camel Route using YAML DSL",
+        "category": "Kaoto",
+        "enablement": "workspaceFolderCount != 0"
+      },
+      {
+        "command": "kaoto.camel.jbang.routes.kamelet.yaml",
+        "shortTitle": "Create a Kamelet",
+        "title": "Create a Kamelet using YAML DSL",
+        "category": "Kaoto",
+        "enablement": "workspaceFolderCount != 0"
+      },
+      {
+        "command": "kaoto.camel.jbang.routes.pipe.yaml",
+        "shortTitle": "Create a Pipe",
+        "title": "Create a Custom Resource Pipe using YAML DSL",
+        "category": "Kaoto",
+        "enablement": "workspaceFolderCount != 0"
+      },
+      {
+        "command": "kaoto.camel.jbang.project.quarkus.new",
+        "title": "Create a Camel Quarkus project",
+        "category": "Kaoto",
+        "enablement": "workspaceFolderCount != 0"
+      },
+      {
+        "command": "kaoto.camel.jbang.project.springboot.new",
+        "title": "Create a Camel on SpringBoot project",
```

Comment:
Noticed that we have the error on our other extension but it is Spring Boot with a space